openmicroscopy/openmicroscopy
# Bump Bio-Formats version to 5.3.0-m2

This commit should update the version of the Bio-Formats library consumed by OMERO to use 5.3.0-m2. This should contain no breaking change but should include the component rearchitecture work as well as the JPEG-XR support for CZI.

To test this PR:
- check the builds including all integration tests remain green
- check the included Bio-Formats version is 5.3.0-m2 (e.g. using `bin/omero import -f`)
- check a JPEG-XR compressed CZI file can be imported into the server
